apps/gp2-server/src/data-providers/contentful/utils/announcements.ts

```diff
@@ -11,11 +11,12 @@ export const parseAnnouncements = (announcements: Announcements) =>
     .filter(
       (announcement): announcement is AnnouncementItem => announcement !== null,
     )
-    .map((announcement: AnnouncementItem) => {
-      const deadline = DateTime.fromISO(announcement.deadline);
-      return {
-        description: announcement.description ?? '',
-        deadline: deadline.toUTC().toString(),
-        link: announcement.link ?? undefined,
-      };
-    }) || [];
+    .filter(
+      ({ deadline }) => DateTime.fromISO(deadline).toUTC() > DateTime.utc(),
```

Comment:
this ordering can be requested in the query, you just need to send the `sortBy` and `sortOrder`